packages/docusaurus-plugin-content-docs/src/__tests__/frontMatter.test.ts

```diff
@@ -394,3 +394,36 @@ describe('validateDocFrontMatter draft', () => {
     ],
   });
 });
+
+describe('validateDocFrontMatter last_update', () => {
+  testField({
+    prefix: 'last_update',
+    validFrontMatters: [
+      {last_update: undefined},
```

Comment:
do we accept empty objects or reject it?

Reply:
I think it would be better to reject empty objects, as other front matter options like `title` and `authors` throw an error when they are defined but given null.